res/mods/innoxia/items/clothing/template/socks.xml

```diff
@@ -43,18 +43,18 @@
 			- Example: Normal socks could be worn by both males and females without issue, so I set this to "ANDROGYNOUS". -->
 		<femininity>ANDROGYNOUS</femininity> 
 		
-		<!-- The slots that this clothing is able to be fit into. The game only supports up to 4 unique slots, so if you define more than 4, the rest of them won't show up in-game. These definitions preserve ordering, so use the top one for the most common slot to be fitted into. This is especially important for NPCs, as they will use the top slot for determining which slot this clothing should fit into.  Possible slots are found here: https://github.com/Innoxia/liliths-throne-public/blob/master/src/com/lilithsthrone/game/inventory/InventorySlot.java 
+		<!-- The slots that this clothing is able to be fit into. The game only supports up to 4 unique slots, so if you define more than 4, the rest of them won't show up in-game. These definitions preserve ordering, so use the top one for the most common slot to be fitted into. This is especially important for NPCs, as they will use the top slot for determining which slot this clothing should fit into.  Possible slots are found here: https://github.com/Innoxia/liliths-throne-public/blob/dev/src/com/lilithsthrone/game/inventory/InventorySlot.java 
 			- Example: I defined "SOCK" as the top slot, as this is the intended slot for socks. As I also want the player to be able to equip socks onto their hands, I defined "HAND" as a secondary slot.-->
 		<equipSlots>
 			<slot>SOCK</slot>
 			<slot>HAND</slot>
 		</equipSlots>
 		
-		<!-- The rarity of this item. Anything less than EPIC may end up being modified in the code. Possible rarities are found here: https://github.com/Innoxia/liliths-throne-public/blob/master/src/com/lilithsthrone/game/inventory/Rarity.java 
+		<!-- The rarity of this item. Anything less than EPIC may end up being modified in the code. Possible rarities are found here: https://github.com/Innoxia/liliths-throne-public/blob/dev/src/com/lilithsthrone/game/inventory/Rarity.java 
 			- Example: Socks are most definitely "COMMON", but I've defined these as "LEGENDARY" so as to prevent them from naturally spawning in the game.-->
 		<rarity>LEGENDARY</rarity> 
 		
-		<!-- The set that this clothing belongs to. I will add a way to define your own sets, but for now, existing sets can be found here: https://github.com/Innoxia/liliths-throne-public/blob/master/src/com/lilithsthrone/game/inventory/clothing/ClothingSet.java
+		<!-- The set that this clothing belongs to. I will add a way to define your own sets, but for now, existing sets can be found here: https://github.com/Innoxia/liliths-throne-public/blob/dev/src/com/lilithsthrone/game/inventory/clothing/ClothingSet.java
```

Comment:
It's been brought to my attention by https://discord.com/channels/302617912949211136/446038411791433729/1061352284489003188 that this referenced file no longer exists at all. I think all vanilla set bonuses were moved to xml at some point, under res/setBonuses